tap_asana/streams/sections.py

```diff
@@ -23,10 +23,17 @@ def get_objects(self):
     session_bookmark = bookmark
     modified_since = bookmark.strftime("%Y-%m-%dT%H:%M:%S.%f")
     opt_fields = ",".join(self.fields)
+
+    # collect list of project ids
+    project_ids = []
     for workspace in self.call_api("workspaces"):
       for project in self.call_api("projects", workspace=workspace["gid"]):
-        for section in Context.asana.client.sections.get_sections_for_project(project_gid=project["gid"], owner="me", opt_fields=opt_fields):
-          yield section
+        project_ids.append(project["gid"])
```

Comment:
How big this list will be? If is a big list this can potentially be an issue as you keep appending.

Reply:
It depends from client to client. But, for the safe side, we have added the yielding of project ids.